test/test_other.py

```diff
@@ -13308,6 +13308,25 @@ def test_em_js_deps(self):
     ''')
     self.do_runf('f2.c', emcc_args=['f1.c'])
 
+  def test_em_js_deps_cpp(self):
+    # Check that EM_JS_DEPS is not mangled in C++.
```

Comment:
Update comment and maybe test title to refer to anon namespace issue.

Reply:
Done